crates/hdk/src/lib.rs

```diff
@@ -2,16 +2,21 @@
 //!
 //! Functions of a Holochain application (hApp) can be organized into reusable components. In Holochain terminology these components are called "zomes".
 //! One or multiple zomes are compiled into a WebAssembly (WASM) binary, referred to as a DNA. All of the DNAs of an application are bundled to a hApp.
+//! In short, that structure is __hApp -> DNA -> zome -> function__.
+//!
+//! hApps are required to produce and validate data deterministically. All data is stored in a content-addressable manner, retrieved by hash value.
```

Comment:
Mmm not all data is stored in a content-addressable manner. Links and CRUD metadata is not. It is true though that only content-addressable data can be queried at validation time.

Reply:
@guillemcordoba Thanks for pointing that out. I've removed that phrase, I don't feel it's relevant in this paragraph.